<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CR5GoodsReservationsFilter</class>
 <widget class="QWidget" name="CR5GoodsReservationsFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="C5DateEdit" name="leEnd"/>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>139</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>271</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="C5DateEdit" name="leStart"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Start date</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>End date</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Search date</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbDateOption">
     <item>
      <property name="text">
       <string>Show current reserves</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date created</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date expired</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date completed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date canceled</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
